component/header.jsx

```diff
@@ -65,6 +65,9 @@ const Nav = ({ path }) => (
       </LinkMatch>
       {' - '}
       <LinkMatch path={path} href="/profile">Profile</LinkMatch>
+       {' - '}
+      <LinkMatch path={path} href="/studentlist">Liste des etudiants</LinkMatch>
```

Comment:
for text everything is in english so far, if we want one day to use the platform outside of ivory coast and also to familiarize students with english:
My reasoning is this, there will be some part in english may it be content or code.
As such the only way to be consistent is to only use english.